pyflakes/checker.py

```diff
@@ -989,6 +989,22 @@ def ignore(self, node):
         EQ = NOTEQ = LT = LTE = GT = GTE = IS = ISNOT = IN = NOTIN = \
         MATMULT = ignore
 
+    def RAISE(self, node):
+        self.handleChildren(node)
+
+        # Check for `
+        raise_arg = node.exc
+        if not isinstance(raise_arg, ast.Call):
```

Comment:
What about `raise NotImplemented`?

Reply:
good point. I can make this look for an `ast.Name` with an id of "NotImplemented" as well.